trecs/components/users.py

```diff
@@ -514,11 +513,41 @@ def get_user_feedback(self, *args, **kwargs):
             self.compute_user_scores(item_attributes)
         # record interactions if needed to ensure users don't repeat interactions
         if not self.repeat_interactions:
+            # set scores back to the original scores
```

Comment:
Given that a user might read "original_scores" as equivalent to non-normalized scores, maybe you could use a different variable name that reflects what this transformation is--that (if I understand correctly) what it is doing is replacing the `Inf` placeholders with the actual user scores.

Reply:
Gotcha! i changed the name to `prev_interacted_scores`